lxd/instance/drivers/driver_qemu.go

```diff
@@ -975,6 +975,11 @@ func (vm *qemu) RegisterDevices() {
 	return
 }
 
+// SaveConfigFile is not used by VMs.
+func (vm *qemu) SaveConfigFile() error {
+	return fmt.Errorf("SaveConfigFile Not Implemented")
```

Comment:
Should we have a generic `NotImplemented` we can use for such things?

Reply:
@stgraber yep will do, there's a few other places I can add that too.